sigmadsp/hardware/adau14xx.py

```diff
@@ -137,3 +143,21 @@ def adjust_volume(self, adjustment_db: float, address: int) -> float:
         )
 
         return linear_to_db(new_volume)
+
+    def safeload(self, address: int, data: bytes, count: int = 1):
```

Comment:
ah, so it does have weird surprises then... I have no means of testing this, but https://www.analog.com/media/en/technical-documentation/data-sheets/ADAU1452_1451_1450.pdf doesn't seem to mention it (the software features chapter, page 93):

> The address_SafeLoad parameter is the target address in parameter RAM. This designates the first address to be written in the safeload transfer. If more than one word is written, the address increments automatically for each data-word. 
>
> The num_SafeLoad parameter designates the number of words to be written. For a biquad filter algorithm, the number of words to be written is five because there are five coefficients in a biquad IIR filter. For a simple mono gain algorithm, the number of words to be written is one. This parameter also serves as the trigger; when it is written, a safeload write is triggered on the next frame.

Reply:
Ok, I am not sure. I have to read more about the topic.